modules/doc/content/application_development/build_system.md

```diff
@@ -44,80 +49,88 @@ some of the environment variables you can set and their effects:
 `src/kernels` you would be likely to find 10-15 Kernels that all inherit from `Kernel.h`.  However,
 if we were to compile all of these files separately (as is commonly done) each of the `#include`
-trees would have to be found and compiled for each one of these files individually.  In addition,
+trees would have to be found and compiled for each one of these files individually. In addition,
 each of these `.C` files would create a separate `.o` file that would ultimately need to be linked
 into the final library / application.
+!style-end!
 
 A different idea is to compile all of these files together, providing considerable savings for
 evaluating the `#include`s and ultimately just producing _one_ `.o` file to be linked.  This can
-provide a huge savings in compile time.  The following is showing timing for `make -j 16` for various
-sets of applications/modules/framework on the Falcon computer at Idaho National Laboratory.  It is
-plain to see how much time is saved by using both header symlinking and unity builds.
+provide a huge savings in compile time. The following is showing timing for `make -j 16` for
+various sets of applications/modules/framework on the Falcon computer at Idaho National Laboratory.
+It is plain to see how much time is saved by using both header symlinking and unity builds.
 
 !media media/application_development/falcon_compile_speed.png
         style=width:50%;float:right;margin-left:20px;
        caption=Compile speed on Falcon.
 
-The way we achieve this in MOOSE is using the idea of "Unity Builds".  A Unity Build compiles
-multiple `.C` files together by create a `.C` file that `#include`s all of the other `.C` files.  In
+The way we achieve this in MOOSE is using the idea of "Unity Builds". A Unity Build compiles
+multiple `.C` files together by create a `.C` file that `#include`s all of the other `.C` files. In
 MOOSE this happens in a directory called `build/unity_src` in each application / module.
 
-By default this behavior is +on+.  To turn it off you can set `MOOSE_UNITY=false` in your
-environment.  Further, an application can disable it permanently by putting `MOOSE_UNITY := false` at
-the top of their Makefile.
+By default this behavior is +on+. To turn it off you can set `MOOSE_UNITY=false` in your
+environment.  Further, an application can disable it permanently by putting `MOOSE_UNITY := false`
+at the top of their Makefile.
 
-In addition you can also keep from unity building individual directories within your project.  By
-default `src` and `src/base` are not unity built because they generally contain a mix of objects that
-don't benefit from being built simultaneously.  To add to that list, set the `app_non_unity_dirs`
-variable in your `Makefile` before the `include` of `app.mk` like so:
+In addition you can also keep from unity building individual directories within your project. By
+default `src` and `src/base` are not unity built because they generally contain a mix of objects
```

Comment:
```suggestion
default `src` and `src/base` are not built this way because they generally contain a mix of objects
```

Reducing awkward phrasing since what "this" is is clear from the previous sentence.